docs/src/main/paradox/scala/http/client-side/client-transport.md

```diff
@@ -37,14 +37,17 @@ method.
 ### TCP
 
 The default transport is `ClientTransport.TCP` which simply opens a TCP connection to the target host.
- 
+
 ### HTTPS Proxy
 
-A transport that connects to target servers via an HTTPS proxy. An HTTPS proxy uses the HTTP `CONNECT` method (as 
-specified in [RFC 7231 Section 4.3.6](https://tools.ietf.org/html/rfc7231#section-4.3.6)) to create tunnels to target 
+A transport that connects to target servers via an HTTPS proxy. An HTTPS proxy uses the HTTP `CONNECT` method (as
+specified in [RFC 7231 Section 4.3.6](https://tools.ietf.org/html/rfc7231#section-4.3.6)) to create tunnels to target
 servers. The proxy itself should transparently forward data to the target servers so that end-to-end encryption should
 still work (if TLS breaks, then the proxy might be fussing with your data).
 
+This approach is commonly used to securely proxy requests to HTTPS endpoints. In theory it could also be used to proxy
+requests targeting HTTP endpoints, but we have not yet found a proxy that in fact allows this.
```

Comment:
By which means has the server prevented it? Does it also mean that most proxies don't allow connecting to arbitrary ports?